### PR TITLE
added command line arguments - fixes #10

### LINKED ISSUE #10: --server option
Something like:

```bash
$ xml2kinto --server https://kinto.dev.mozaws.net/
```

- [x] add a --server option so we can point the script to the right Kinto
- [x] expose the value as an environment variable so it's easy to change when "make sync" runs
- [x] document the option in README.rst for Ops

### PR DESCRIPTION
**Pull request by tarekziade**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -35,7 +37,7 @@ tests-once: install-dev
 
 tests:
 	@rm -fr .coverage
-	tox
+	$(VENV)/bin/tox
```
Comment (Natim) [inline Makefile]: Why you no have tox globally :+1:

File: xml2kinto/__main__.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import os
+import argparse
+
 from xml2kinto.synchronize import synchronize
 
 # options to move to a config file
 xml_file = os.path.abspath(os.path.join(os.path.dirname(__file__),
                                         '..', 'blocklist.xml'))
-auth = ('mark', 'p4ssw0rd')
```
Comment (Natim) [inline xml2kinto/__main__.py]: Please do not change that, it would let use pass a Authorization backend to requests.

File: xml2kinto/__main__.py [hunk 3]
```diff
@@ -12,14 +14,27 @@
...
+def main(args=None):
+    parser = argparse.ArgumentParser(description='Syncs a Kinto DB.')
+
+    parser.add_argument('-s', '--kinto-server', help='Kinto Server',
+                        type=str, default=kinto_server)
+
+    parser.add_argument('-x', '--xml-file', help='XML Source file',
+                        type=str, default=xml_file)
+
+    parser.add_argument('-a', '--auth', help='BasicAuth user:pass',
```
Comment (Natim) [inline xml2kinto/__main__.py]: We should probably access both ``--auth-type`` and ``--auth`` like requests does.
Then if we want to use FxA in production we would be able to.

File: xml2kinto/tests/test_main.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import unittest
```
Comment (Natim) [inline xml2kinto/tests/test_main.py]: Then we should remove __main__.py from the .coveragerc file

File: xml2kinto/__main__.py (hunk 3 above)
Comment (Natim) [inline xml2kinto/__main__.py]: I opened https://github.com/mozilla-services/xml2kinto/issues/12 to handle this case so we can merge this first.

**Comment by Natim** [comment]
r+

File: Makefile (hunk 1 above)
Comment (tarekziade) [inline Makefile]: the Make is supposed to work on a fresh python install, and you are installing tox locally in your makefile, so why not use the tox you installed ?

File: xml2kinto/__main__.py (hunk 2 above)
Comment (tarekziade) [inline xml2kinto/__main__.py]: Sorry I don't understand this comment

File: xml2kinto/__main__.py (hunk 3 above)
Comment (tarekziade) [inline xml2kinto/__main__.py]: We don't add new features for now.

File: Makefile (hunk 1 above)
Comment (Natim) [inline Makefile]: I totally agree with you :+1: good catch

**Pull request merged**